src/main.c

```diff
@@ -65,16 +67,16 @@ int main() {
     SPI_ProtoInit();
     CONFIG_ReadDisplay();
     CONFIG_ReadModel(CONFIG_GetCurrentModel());
-    CONFIG_ReadLang(Transmitter.language);
+    CONFIG_ReadLang(Transmitter.language); //diff15
 
-    BACKLIGHT_Brightness(Transmitter.backlight);
+    BACKLIGHT_Brightness(Transmitter.backlight); //diff16
     LCD_Contrast(Transmitter.contrast);
-    LCD_SetFont(DEFAULT_FONT.font);
+    LCD_SetFont(DEFAULT_FONT.font); //diff17
     LCD_SetFontColor(DEFAULT_FONT.font_color);
 
-    GUI_HandleButtons(1);
+    GUI_HandleButtons(1); //diff18
 
-    MIXER_Init();
+    MIXER_Init(); //diff19
```

Comment:
Lint Error:
main.c:77:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:77:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:79:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:79:  Should have a space between // and comment  [whitespace/comments] [4]